plugin/plugin.go

```diff
@@ -120,56 +105,56 @@ func (p *CLIPlugin) GetMetadata(ctx context.Context, req *proto.GetMetadataReque
 // DescribeKey returns the KeySpec of a key.
 //
 // if ContractVersion is not set, it will be set by the function.
-func (p *CLIPlugin) DescribeKey(ctx context.Context, req *proto.DescribeKeyRequest) (*proto.DescribeKeyResponse, error) {
+func (p *CLIPlugin) DescribeKey(ctx context.Context, req *plugin.DescribeKeyRequest) (*plugin.DescribeKeyResponse, error) {
 	if req.ContractVersion == "" {
 		req.ContractVersion = proto.ContractVersion
```

Comment:
Will we replace `proto.ContractVersion` with `plugin.ContractVersion` in `notation-plugin-framework-go/plugin`?

Reply:
Done